run/local-cluster.sh

```diff
@@ -165,7 +165,7 @@ for ATTEMPT in $(seq 1 $MAX_WAIT); do
     GOSSIP_URL="$DOCKERHOST:${HTTP_PORTS[$i]}/_status/gossip"
     GOSSIP=$(curl --noproxy '*' -s $GOSSIP_URL)
     for j in $(seq 1 $((2*NODES))); do
-      if [[ ! -z $(echo $GOSSIP | grep "node-$j") ]]; then
+      if [[ ! -z $(echo $GOSSIP | grep -E "node.$j") ]]; then
```

Comment:
I thought it was a colon separator. With grep -E, you're allowing any character between "node" and the node ID. Why not just grep "node:$j"?